specs/experimental/custom-gas-token.md

```diff
@@ -161,8 +159,8 @@ function depositERC20Transaction(
 ) public;
 ```
 
-This function MUST revert when `ether` is the L2's native asset. It MUST not be `payable`, meaning that it will
-revert when `ether` is sent with a `CALL` to it. It uses a `transferFrom` flow, so users MUST first `approve()`
+This function MUST revert when `Ether` is the L2's native asset. It MUST not be `payable`, meaning that it will
+revert when `Ether` is sent with a `CALL` to it. It uses a `transferFrom` flow, so users MUST first `approve()`
```

Comment:
I think its ok to not capitalize every instance of `ether` - ie bring up chisel and type `1 ether` and see that it works while `1 Ether;` doesn't work. The only place we need to capitalize it is in the `name()` getter